src/escalus_auth.erl

```diff
@@ -258,8 +258,9 @@ scram_sha_validate_server(HashMethod, SaltedPassword, AuthMessage, ServerSignatu
             false
     end.
 
-scram_sha_auth_payload(none) -> <<"y,,">>;
-scram_sha_auth_payload(tls_unique) -> <<"p=tls-unique,,">>.
+scram_sha_auth_payload(ssl, _) -> <<"n,,">>;
+scram_sha_auth_payload(fast_tls, none) -> <<"y,,">>;
```

Comment:
So to get escalus login with `y` flag we'd need to have fast_tls as tls_module and try to login with scram mechanism that is not doing channel binding?

Reply:
@janciesla8818 exactly. Something like this one would do for example:
```
Specs = [..., {tls_module, fast_tls}, {auth, {escalus_auth,auth_sasl_scram_sha256}}].
```

Reply:
Cool, in such case I think I won't event have to change my tests for the other PR.